### PR TITLE
Add new data scrapper

### PR DESCRIPTION
**Pull request by pp1e**

### CODE REVIEW

File: config/scrapper_config_example.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Message broker server URL. You can use any broker,
+# such as RabbitMQ or Redis.
+# This is an example of RabbitMQ default URL.
+MESSAGE_BROKER_URL = 'amqp://guest:guest@localhost//'
```
Comment (AlexSkvorz) [inline config/scrapper_config_example.py:4]: I think it won't work, because in the end of url //

File: data_scrapper/data_scrapping.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+from bs4 import BeautifulSoup
+
+from config import scrapper_config
+from data_scrapper.calculations import calculate_week_user_profits
+from old_database import users_rights_table, weeks_stats_table
+from utils import date_utils
+
+
+@dataclass
+class MonitorData:
```
Comment (AlexSkvorz) [inline data_scrapper/data_scrapping.py:16]: move this to another file. This file looks oversaturated

File: data_scrapper/data_scrapping.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    return MonitorData(
+        overall_balance=dollars_to_number(soup.find('a', id='17542800balance').text),
+        overall_profit=dollars_to_number(soup.find('span', id='17542800profit_total').text),
+        week_profit=dollars_to_number(soup.find('span', id='17542800profit_w').text),
+        overall_profit_percent=percents_to_number(soup.find('span', id='17542800profit_total_pr').text),
+        week_profit_percent=percents_to_number(soup.find('span', id='17542800profit_w_pr').text),
+    )
+
+
+def get_current_week_profits(db_connection, actual_overall_balance):
```
Comment (AlexSkvorz) [inline data_scrapper/data_scrapping.py:67]: this function and functions below depend on database, I think should move in another file

File: config/scrapper_config_example.py (hunk 1 above)
Comment (pp1e) [reply config/scrapper_config_example.py:4]: '//' in this URI specify a virtual host by default

Comment (AlexSkvorz) [reply config/scrapper_config_example.py:4]: Also, I'd put {username}:{password}@{ip} in config file

**Pull request merged**